consensus/src/execution_ctx.rs

```diff
@@ -156,15 +157,29 @@ impl<'a, T: Operations + 'static, DB: Database> ExecutionCtx<'a, T, DB> {
                         self.process_inbound_msg(phase.clone(), msg).await
                     {
                         if open_consensus_mode {
+                            info!(
+                                "Received message in open consensus: {:?}",
+                                step_result.topic()
+                            );
+                            if let Payload::Quorum(q) = &step_result.payload {
+                                if !q.att.result.failed() {
+                                    info!("Sending quorum in open consensus",);
```

Comment:
Can we use a single "keyword" for any log related to open_consensus?
e.g 

`info!( mode = "open_consensus", event = "send quorum",  hash = )`

Also, `info!(event = "run event_loop in open_mode", ?dur);` this one should be updated